numpy/random/bit_generator.pyx

```diff
@@ -294,6 +294,27 @@ cdef class SeedSequence():
     >>> sq2 = np.random.SeedSequence(sq1.entropy)
     >>> np.all(sq1.generate_state(10) == sq2.generate_state(10))
     True
+    
+    Examples
+    --------
+    
+    Here we construct a `PCG64` generator using a predefined seed, 1234
```

Comment:
I also don't think that 1234, or any "common" seed, e.g., 0, should appear in the docs.